### ISSUE TITLE
Error after upgrading to v3.3.1

### ISSUE THREAD

**Issue by Croccifixio** [cascornelissen/svg-spritemap-webpack-plugin#75]
After upgrading to v3.3.1, the plugin throws the following error when building with webpack:

```
C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\webpack-cli\bin\cli.js:265
                                throw err;
                                ^

Error: must provide pattern
    at new GlobSync (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\glob\sync.js:31:11)
    at Function.globSync [as sync] (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\glob\sync.js:26:10)
    at directories.inputOptions.patterns.reduce (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\svg-spritemap-webpack-plugin\lib\index.js:321:38)
    at Array.reduce (<anonymous>)
    at SVGSpritemapPlugin.updateDependencies (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\svg-spritemap-webpack-plugin\lib\index.js:319:50)
    at SyncHook.eval [as call] (eval at create (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:7:1)
    at SyncHook.lazyCompileHook (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\tapable\lib\Hook.js:154:20)
    at webpack (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\webpack\lib\webpack.js:51:30)
    at processOptions (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\webpack-cli\bin\cli.js:256:16)
    at yargs.parse (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\webpack-cli\bin\cli.js:373:3)
    at Object.parse (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\yargs\yargs.js:567:18)
    at C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\webpack-cli\bin\cli.js:49:8
    at Object.<anonymous> (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\webpack-cli\bin\cli.js:375:3)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (C:\Users\Croccifixio\Desktop\playground\sprite-map-pr\node_modules\webpack\bin\webpack.js:156:2)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ build: `webpack`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Croccifixio\AppData\Roaming\npm-cache\_logs\2019-05-17T22_44_12_123Z-debug.log
```

I have created a [repo](https://github.com/Croccifixio/svg-spritemap-webpack-plugin-issue) where the error can be reproduced. The following webpack config is used:

```js
const SVGSpritemapPlugin = require('svg-spritemap-webpack-plugin')

module.exports = {
  entry: './index.js',
  plugins: [
    new SVGSpritemapPlugin('icons/*.svg', {
      output: {
        filename: "dist/icons.svg",
      },
    }),
  ],
}
```

Downgrading to v3.3.0 causes the build to proceed without any errors.

---

Windows version: 10.0.17763 Build 17763
Node version: 10.15.3
svg-spritemap-webpack-plugin version: 3.3.1
webpack version: 4.31.0

**Comment by cascornelissen** [comment]
Thank for creating the issue, this is related to #74 and the current workaround is to make sure you use `path.resolve` around the first argument. So `path.resolve('icons/*.svg')` in your case.

**Comment by cascornelissen** [comment]
@Croccifixio, a fix for this bug has been released in version [`3.3.2`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v3.3.2) 🚀
